src/Service/Cases.php

```diff
@@ -96,4 +96,52 @@ public function getByCaseId(string $caseId, string $caseTypePrefix = null): ?arr
 
         return $cases['CasesInfo'][0] ?? null;
     }
+
+    /**
+     * Example query.
+     *
+     * $query = [
+     *   'CaseTypePrefix' => 'BOR',
+     *   'MetadataXml' =>
+     *     "<z:row xmlns:z=\"#RowsetSchema\"
+     *      ows_Title=\"0123456789 - Test borger\"
+     *      ows_CCMContactData=\"Test borger;#;#0123456789;#;#\"
+     *      ows_CCMContactData_CPR=\"0123456789\"
+     *      ows_CaseStatus=\"Åben\"
+     *      />",
+     *  'ReturnWhenCaseFullyCreated' => true,
+     * ];
+     */
+    public function CreateCase(array $query): ?array
```

Comment:
The parameter should not be called `$query`, but rather `$data` or something similar. “Query” is used for querying and not for creating stuff.

Reply:
Updated variable name.